analyzers/rspec/cs/S2955.html

```diff
@@ -27,7 +27,7 @@ <h4>Compliant solution</h4>
 <pre data-diff-id="1" data-diff-type="compliant">
 bool IsDefault&lt;T&gt;(T value)
 {
-  if(object.Equals(value, default(T)))
+  if (EqualityComparer&lt;T&gt;.Default.Equals(value, default(T)))
```

Comment:
This change has been done by @gregory-paidis-sonarsource in the context of https://github.com/SonarSource/rspec/pull/3794.
I have added this rule to the list of rules changed without RSPEC update, in [the issue](https://github.com/SonarSource/sonar-dotnet/issues/8913).